BootloaderCommonPkg/Include/Library/IasImageLib.h

```diff
@@ -66,6 +66,7 @@ typedef struct {        /* a file (sub-image) inside a boot image */
   VOID                 *Addr;
   UINT32                Size;
   IMAGE_ALLOCATE_TYPE   AllocType;
+  UINT32                Name;
```

Comment:
Please add a comment here that this is the name specified for the component when building the container.

Reply:
Added